### ISSUE TITLE
[Request] Allow QField label-width of 12

### ISSUE THREAD

**Issue by dboune** [quasarframework/quasar#1388]
In some cases is is preferable to have a QField label located above the field on desktop, as it is displayed on mobile. One such example is with the QOptionGroup component.

An easy way to do this on a singular basis is to set `label-width` to 12. The result appears as expected, but a vue warning is thrown because the custom validator assigned to the property expects a value between 1 and 12.

```html
<q-field label="Options" :label-width="12">
  <q-option-group .../>
</q-field>
```

**Example Work-Around** _(if the warning is undesirable)_

```html
<q-field class="full-width-label" label="Options">
  <q-option-group .../>
</q-field>
```

```stylus
.full-width-label .q-field-label
    max-width 100%
    flex 0 0 100%
```

**Comment by rstoenescu** [comment]
Available in future v0.15

**Comment by rstoenescu** [comment]
Examples:
```html
<!-- always vertical -->
<q-field orientation="vertical" label="Some label">...</q-field>
<q-field label-width="12" label="Some label">...</q-field>

<!-- always horizontal -->
<q-field orientation="horizontal" label="Some label">...</q-field>
```

Note that "label-width" will support numbers in String format too.
